build/commands/lib/config.js

```diff
@@ -565,6 +572,13 @@ Config.prototype.buildArgs = function () {
     args.symbol_level = 1
   }
 
+  if (this.isCoverage()) {
+    args.symbol_level = 1
+    args.use_clang_coverage = true
+    args.use_clang_profiling_inside_sandbox = true
+    args.coverage_instrumentation_input_file = '//out/files-to-instrument.txt'
```

Comment:
based on the comments for `coverage_instrumentation_input_file` this should go in the relevant output directory (e.g. //out/Release/coverage_instrumentation_input.txt)